### ISSUE TITLE
[BUG] Missing Release of Resource after Effective Lifetime [High Severity]

### ISSUE THREAD

**Issue by ocasiom** [exceljs/exceljs#2608]
## 🐛 Bug Report

There's a snyk issue with the latest build

Missing Release of Resource after Effective Lifetime [High Severity][https://security.snyk.io/vuln/SNYK-JS-INFLIGHT-6095116] in inflight@1.0.6

Lib version: 4.4.0

Introduced through: exceljs@4.4.0 › archiver@5.3.2 › archiver-utils@2.1.0 › glob@7.2.3 › inflight@1.0.6

**Comment by moodys-onkar** [comment]
Yes, this is also impacting us. Please resolve this at the earliest.

**Comment by chandanjainn** [comment]
+1

**Comment by AshishGuptaNGupta** [comment]
+1

**Comment by raffaele-moodys** [comment]
+1

**Comment by relaxedleaf** [comment]
Is there an update on this?

**Comment by henryfung3a27** [comment]
Try https://github.com/exceljs/exceljs/issues/2829#issuecomment-2421305827
